lib/dfe/analytics.rb

```diff
@@ -143,6 +143,9 @@ def self.allowlist_pii
 
     def self.hidden_pii
       Rails.application.config_for(:analytics_hidden_pii)
+    rescue StandardError
```

Comment:
We normally catch `RuntimeError` for these config errors, as it's a bit less catch all.